test/hotspot/jtreg/compiler/floatingpoint/TestSubNodeFloatDoubleNegation.java

```diff
@@ -53,11 +53,14 @@ public static void assertResults() {
     }
 
     @Test
-    // Match a generic SubNode, because scalar Float16 operations are often
-    // performed as float operations.
-    @IR(counts = { IRNode.SUB, "2" }, applyIfPlatform = {"riscv64", "false"})
-    @IR(counts = { IRNode.SUB, "2" }, applyIfCPUFeature = {"zfh", "false"})
-    @IR(counts = { IRNode.SUB_HF, "2" }, applyIfCPUFeature = {"zfh", "true"})
+    @IR(counts = { IRNode.SUB, "2" }, applyIfPlatform = {"x64", "true"}, applyIfCPUFeature = {"avx512_fp16", "false"})
+    @IR(counts = { IRNode.SUB_HF, "2" }, applyIfPlatform = {"x64", "true"}, applyIfCPUFeature = {"avx512_fp16", "true"})
+    // TODO: uncomment once Float16 support lands in aarch64
```

Comment:
Maybe you can also add the actual issue number:
```suggestion
    // TODO: uncomment once Float16 support lands in aarch64 with JDK-8345125
```